ngap/handler.go

```diff
@@ -635,9 +637,15 @@ func HandleNGSetupRequest(ran *context.AmfRan, message *ngapType.NGAPPDU) {
 
 	if cause.Present == ngapType.CausePresentNothing {
 		ngap_message.SendNGSetupResponse(ran)
+		//send nf(gnb) status notification
+		gnbStatus := mi.MetricEvent{EventType: mi.CNfStatusEvt,
```

Comment:
Isn't it good to have this gNB state in sctplb ?

Reply:
@vthiruveedula : Lets discuss this